docker-compose.yml

```diff
@@ -46,4 +49,51 @@ services:
       timeout: 20s
       retries: 10
       start_period: 40s
-      interval: 30s
\ No newline at end of file
+      interval: 30s
+
+  redis:
+    container_name: redis
+    image: "redis:6.2.4-alpine"
+    networks:
+      - mojaloop-net
+    ports:
+      - "6379:6379"
+    restart: always
+
+  ml-testing-toolkit:
+    image: mojaloop/ml-testing-toolkit:v12.4.0
+    container_name: ml-testing-toolkit
+    volumes:
+      - "./docker/ml-testing-toolkit/spec_files:/opt/mojaloop-testing-toolkit/spec_files"
+      - "./docker/ml-testing-toolkit/secrets:/opt/mojaloop-testing-toolkit/secrets"
+    ports:
+      - "15000:5000"
+      - "5050:5050"
+    command: npm start
+    networks:
+      - mojaloop-net
+    depends_on:
+      - mongo
+    extra_hosts:
```

Comment:
Same as above. We don't need `extra _hosts`, it just understanding the networking more difficult.